test/unit/period/generators/bi-monthly_spec.js

```diff
@@ -14,11 +14,11 @@ describe('Bi-monthly period', () => {
 
     describe('generateBiMonthlyPeriodsForYear()', () => {
         it('should not allow years before the year zero', () => {
-            expect(() => generateBiMonthlyPeriodsForYear(-10)).to.throw('Generator does not support generating year before the year 0');
+            expect(() => generateBiMonthlyPeriodsForYear(-10)).to.throw();
```

Comment:
Any specific reason why we'd remove the error messages? The test is now less solid as it could throw anything and the test would pass.